docs/mem_quick_start.ipynb

```diff
@@ -0,0 +1,597 @@
+    "\n",
+    "# Set the project id\n",
+    "! gcloud config set project {PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Usage"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set AlloyDB database values\n",
+    "Find your database values, in the [AlloyDB cluster page](https://console.cloud.google.com/alloydb?_ga=2.223735448.2062268965.1707700487-2088871159.1707257687)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# @title Set Your Values Here { display-mode: \"form\" }\n",
+    "REGION = \"us-central1\"  # @param {type: \"string\"}\n",
+    "CLUSTER = \"my-alloydb-cluster\"  # @param {type: \"string\"}\n",
+    "INSTANCE = \"my-alloydb-instance\"  # @param {type: \"string\"}\n",
+    "DATABASE = \"my-database\"  # @param {type: \"string\"}\n",
+    "TABLE_NAME = \"message_store\"  # @param {type: \"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### AlloyDBEngine Connection Pool\n",
+    "\n",
+    "One of the requirements and arguments to establish AlloyDB Model endpoint Management is a `AlloyDBEngine` object. The `AlloyDBEngine`  configures a connection pool to your AlloyDB database, enabling successful connections from your application and following industry best practices.\n",
+    "\n",
+    "To create a `AlloyDBEngine` using `AlloyDBEngine.from_instance()` you need to provide only 5 things:\n",
+    "\n",
+    "1. `project_id` : Project ID of the Google Cloud Project where the AlloyDB instance is located.\n",
+    "1. `region` : Region where the AlloyDB instance is located.\n",
+    "1. `cluster`: The name of the AlloyDB cluster.\n",
+    "1. `instance` : The name of the AlloyDB instance.\n",
+    "1. `database` : The name of the database to connect to on the AlloyDB instance.\n",
+    "\n",
+    "By default, [IAM database authentication](https://cloud.google.com/alloydb/docs/connect-iam) will be used as the method of database authentication. This library uses the IAM principal belonging to the [Application Default Credentials (ADC)](https://cloud.google.com/docs/authentication/application-default-credentials) sourced from the environment.\n",
+    "\n",
+    "Optionally, [built-in database authentication](https://cloud.google.com/alloydb/docs/database-users/about) using a username and password to access the AlloyDB database can also be used. Just provide the optional `user` and `password` arguments to `AlloyDBEngine.from_instance()`:\n",
+    "\n",
+    "* `user` : Database user to use for built-in database authentication and login\n",
+    "* `password` : Database password to use for built-in database authentication and login.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: This tutorial demonstrates the async interface."
+   ]
+  },
```

Comment:
Adding this would enable coverage for sync methods as well
```suggestion
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Note:** This tutorial demonstrates the async interface. All async methods have corresponding sync methods."
   ]
  },
```

Reply:
Edit: **Note:** This tutorial demonstrates the async interface. Several async methods have corresponding sync methods.

Reply:
OR add this disclaimer only for Embeddings class's sections.